GLEW/CMakeLists.txt

```diff
@@ -45,13 +44,11 @@ else()
     target_link_libraries(${GLEW_LIBRARY} ${OPENGL_gl_LIBRARY})
   endif()
   set_target_properties(${GLEW_LIBRARY} PROPERTIES
-                        FOLDER "Libraries" VERSION ${GLEW_VERSION})
+                        FOLDER "${PROJECT_NAME}" VERSION ${GLEW_VERSION})
 
   install(TARGETS ${GLEW_LIBRARY}
-    EXPORT ${CMAKE_PROJECT_NAME}Targets
+    EXPORT ${PROJECT_NAME}Targets
```

Comment:
No, this needs to be CMAKE_PROJECT_NAME. The export set somehow works on a full build level, i.e., when a downstream project does install(EXPORTS) it needs all its dependencies in its export set.